src/amber/cli/generators.cr

```diff
@@ -27,23 +27,21 @@ module Amber::CLI
       @fields = fields
     end
 
-    def generate(command : String, options = nil)
-      if command == "app"
-        if options
-          info "Rendering App #{name} in #{directory}"
-          App.new(name, options.d, options.t).render(directory)
-          unless options.no_deps?
-            info "Installing Dependencies"
-            Helpers.run("cd #{directory} && shards update")
-          end
-        end
+    def generate_app(options)
+      info "Rendering App #{name} in #{directory}"
+      App.new(name, options.d, options.t).render(directory, list: true, interactive: !options.assume_yes?, color: options.no_color?)
```

Comment:
> `!options.assume_yes?`

Maybe this is evidence that there's some drift between interfaces with `interactive` and `options`

Reply:
We default to `false` so I think this makes sense.  `apt-get` uses this `-y` and `--assume-yes` flags, so it seemed to match.

Reply:
Yeah, not a to-do, just an observation that some tech debt might be stacking